.github/workflows/fast.yml

```diff
@@ -103,12 +117,13 @@ jobs:
     - name: Install cargo-fuzz
       if: steps.cargo-fuzz-cache.outputs.cache-hit != 'true'
       run: cargo +nightly install cargo-fuzz --version ${{ env.FUZZ_VERSION }}
-    - name: Build codec
-      run: cargo +nightly fuzz build --fuzz-dir codec/fuzz
-    - name: Build cryptography
-      run: cargo +nightly fuzz build --fuzz-dir cryptography/fuzz
-    - name: Build stream
-      run: cargo +nightly fuzz build --fuzz-dir stream/fuzz
+    - name: Test all targets
```

Comment:
Run fuzz tests for a short duration to make sure they actually work (rather than just compile).